.github/scripts/update-readable-css.sh

```diff
@@ -35,7 +35,7 @@ else
     curl -s "https://codeberg.org/Freedom-to-Write/readable.css/raw/tag/${TAG_VERSION}/readable.css" > static/css/readable.css
     curl -s "https://codeberg.org/Freedom-to-Write/readable.css/raw/tag/${TAG_VERSION}/readable.min.css" > static/css/readable.min.css
     VERSION=$(echo "${TAG_VERSION}" | sed 's/v//')
-    sed -i "s/readable.min.css?v=.*/readable.min.css?v=${VERSION}\">/" layouts/partials/head.html
+    sed -i "s/readable.min.css?v=${CURRENT_VERSION}/readable.min.css?v=${VERSION}/" layouts/partials/head.html
```

Comment:
That's a great idea. I feel a bit stupid now. 😅

But be careful, that the CURRENT_VERSION and TAG_VERSION start with a 'v'. That's why we use a different VERSION in this replacement.

An easy way to fix this is something like `CURRENT_VERSION=$(echo $CURRENT_VERSION | sed 's/v//')`.

An alternative would be to never add the 'v' prefix to CURRENT_VERSION in the first place and prefix it explicitly where it's needed.